mbus-api/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use async_trait::async_trait;
+use bytes::Bytes;
+use mbus_nats::{BusResult, BusSubscription};
+use serde::{de::DeserializeOwned, Serialize};
+
+mod common;
+pub mod mbus_nats;
+pub mod message;
+
+pub trait NatsMessage: Send + Sync {
```

Comment:
Do we really want to tie ourselves to Nats? What about making this generic, in case we need to replace nats with something else?